query_optimizer/tests/execution_generator/Select.test

```diff
@@ -953,19 +953,79 @@ WHERE double_col < 0
 ==
 
 # Window Aggregation Test.
-# Currently this is not supported, an empty table will be returned.
-SELECT avg(int_col) OVER w FROM test
+SELECT char_col, long_col, avg(long_col) OVER w FROM test
 WINDOW w AS
-(PARTITION BY char_col
- ORDER BY long_col DESC NULLS LAST
+(ORDER BY char_col DESC NULLS LAST
```

Comment:
Ah I get what you are saying. In that case we don't actually need to do a sort since nothing will be changed, and the sort attribute will be empty (though now we don't pass the vector into the handle).

The sort attribute is not needed currently since we only support ROWS mode. To support RANGE mode, we have to add sort attribute to calculate the window frame. They will be introduced in the future PR.

Thanks for pointing that out :)